# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/test/browser/browser.toml
# Commit: d2381f88168a
# Full Hash: d2381f88168ae15ad51e03a193c81f0267b95b7c
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2024-12-17 17:00:49
# Regressor Bug: 1915355
# File Overlap Count: 1
# Description:
#   Bug 1915355 - Part 6: Add tests. r=cookie-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223386
# ==============================================================================

diff -r 675785853d7a -r d2381f88168a netwerk/cookie/test/browser/browser.toml
--- a/netwerk/cookie/test/browser/browser.toml	Tue Dec 17 08:57:31 2024 +0000
+++ b/netwerk/cookie/test/browser/browser.toml	Tue Dec 17 08:57:31 2024 +0000
@@ -6,6 +6,9 @@
   "head.js"
 ]
 
+["browser_3pcb_expection.js"]
+support-files = ["setFetchCookie.sjs"]
+
 ["browser_broadcastChannel.js"]
 
 ["browser_chips_partition_cap_child.js"]
@@ -13,6 +16,9 @@
 ["browser_cookie_chips.js"]
 support-files = ["chips.sjs"]
 
+["browser_cookie_empty_name_value.js"]
+support-files = ["cookie_empty_name_value.sjs"]
+
 ["browser_cookie_insecure_overwrites_secure.js"]
 
 ["browser_cookie_purge_sync.js"]
@@ -34,9 +40,6 @@
 ["browser_oversize.js"]
 support-files = ["oversize.sjs"]
 
-["browser_cookie_empty_name_value.js"]
-support-files = ["cookie_empty_name_value.sjs"]
-
 ["browser_partitionedConsole.js"]
 support-files = ["partitioned.sjs"]
 